Scoop/Controller/SearchInitialViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SearchInitialViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 4/10/22.
+//
+
+import UIKit
+
+protocol SearchInitialViewControllerDelegate: AnyObject {
+    func didSelectLocation(viewController: UIViewController, location: String)
+}
+
+class SearchInitialViewController: UIViewController, UISearchResultsUpdating, UISearchBarDelegate {
+    
+    weak var delegate: SearchInitialViewControllerDelegate?
+    
+    private var locationController: LocationViewController!
+    private var searchController = UISearchController()
+    
+    func updateSearchResults(for searchController: UISearchController) {
+        let searchBar = searchController.searchBar
+        guard let searchText = searchBar.text else{
```

Comment:
add space after else and do in one line
`guard let searchText = searchBar.text else { return }`

also since you don't use `searchBar` again, you can just do 
`guard let searchText = searchController.searchBar.text else { return }`